.travis.yml

```diff
@@ -24,12 +24,16 @@ install:
 before_script:
     - export PATH=$HOME/gopath/bin:$PATH
 
-script:
-    - make .gitvalidation
-    - make verify
-    - make binaries
-    - make test
-    - make test-cri
+jobs:
+  include:
+    - stage: Static Check and Build
+      script: 
+        - make .gitvalidation
+        - make verify
```

Comment:
really odd output

I see a build output with make being called in the log. But I don't see the call to make in the travis.yml

I can't see any output from the stages either?